test/parallel/foundationMembers.test.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ *
+ * @group governance
+ */
+
+import { getApi, initApi } from "../../utils/api";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { Assets } from "../../utils/Assets";
+import { User } from "../../utils/User";
+import {
+  getEventErrorFromSudo,
+  getEventResultFromMangataTx,
+} from "../../utils/txHandler";
+import { ExtrinsicResult } from "../../utils/eventListeners";
+import { BN_HUNDRED, BN_THOUSAND } from "gasp-sdk";
+import { Council } from "../../utils/Council";
+import { FoundationMembers } from "../../utils/FoundationMembers";
+
+let testUser: User;
+let foundationAddress: any;
+
+beforeAll(async () => {
+  try {
+    getApi();
+  } catch (e) {
+    await initApi();
+  }
+  await setupApi();
+});
+
+beforeEach(async () => {
+  [testUser] = setupUsers();
+  await Sudo.batchAsSudoFinalized(Assets.mintNative(testUser));
+  foundationAddress = (await FoundationMembers.getFoundationMembers())[0];
+});
+
+test("A founder can update his own foundation address", async () => {
+  await Sudo.asSudoFinalized(
+    Sudo.sudoAsWithAddressString(
+      foundationAddress,
+      FoundationMembers.changeKey(testUser),
+    ),
+  ).then((events) => {
+    const res = getEventResultFromMangataTx(events);
+    expect(res.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  });
+  const foundationMembers = await FoundationMembers.getFoundationMembers();
+  expect(foundationMembers).toContain(testUser.keyRingPair.address);
+});
+
+test("Council can not execute this extrinsic", async () => {
```

Comment:
This test requires : 

1. Create a council
2. Create a proposal
3. Add on this proposal the changeKey
4. Vote this proposal for all the council memebers
5. Close the proposal with all ayes' 
6. Check that the proposal is not executed.

Reply:
Fixed